server/backend/src/users/users.controller.ts

```diff
@@ -21,6 +22,10 @@ export type GetGroupsOutput = {
   }[];
 }[];
 
+export type PutCameraResponse = {
```

Comment:
nit, see my other comment
```suggestion
export type RegisterCameraResponse = {
```